scripts/update-minor.sh

```diff
@@ -0,0 +1,15 @@
+LAST_TAG="$$TRAVIS_TAG"
+
+if [ $LAST_TAG != "" ]; then
+
+MAYOR=$(echo $LAST_TAG | cut -d'.' -f1 | tr -d '[a-z][A-Z]')
+MINOR=$(echo $LAST_TAG | cut -d'.' -f2)
+PATCH=$(echo $LAST_TAG | cut -d'.' -f3)
+
+echo "Current tag: $LAST_TAG"
+
+REPLACEMENT="VERSION $MAYOR.$MINOR.$PATCH"
+echo "Updating CMAKE version: VERSION 1.0.0 -> $REPLACEMENT"
+sed -i.bak "s/VERSION 1.0.0/$REPLACEMENT/g" CMakeLists.txt
```

Comment:
As commented IRL, it's ok to use the version number as placeholder, we should just use `0.0.0` since we'll be in a pre-1.0 at the beginning.